src/reactor/poll_evented.rs

```diff
@@ -275,7 +270,7 @@ impl<E> PollEvented<E> {
 impl<E: Read> Read for PollEvented<E> {
     fn read(&mut self, buf: &mut [u8]) -> io::Result<usize> {
         if let Async::NotReady = self.poll_read() {
-            return Err(io::ErrorKind::WouldBlock.into())
+            return Err(io::ErrorKind::WouldBlock.into());
```

Comment:
Similar with the trailing commas, could stylistic changes like this be left out for now?